I'm probably doing something wrong, but when I try to build the package with ` nix-build -E 'let pkgs = import <nixpkgs> { }; in pkgs.callPackage ./default.nix {}'`, I get:

```
[INFO] --- compiler:3.11.0:compile (default-compile) @ cryptomator ---
[WARNING] Can't extract module name from fuse-nio-adapter-3.0.0.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-linux-aarch64-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-win-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-linux-amd64-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-api-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-mac-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-0.5.1.jar: Unsupported major.minor version 64.0
[INFO] Required filename-based automodules detected: [commons-cli-1.5.0.jar, javax.inject-1.jar]. Please don't publish this project to a public artifact repository!
[INFO] Changes detected - recompiling the module! :source
[INFO] Compiling 322 source files with javac [debug release 20 module-path] to target/classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.105 s
[INFO] Finished at: 2023-09-28T13:36:53Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project cryptomator: Fatal error compiling: error: release version 20 not supported -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

error: builder for '/nix/store/cs23i4af8hzi55s06bd95hqx7gpm08ni-cryptomator-1.10.1.drv' failed with exit code 1;
       last 10 log lines:
       > [INFO] Finished at: 2023-09-28T13:36:53Z
       > [INFO] ------------------------------------------------------------------------
       > [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project cryptomator: Fatal error compiling: error: release version 20 not supported -> [Help 1]
       > [ERROR]
       > [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
       > [ERROR] Re-run Maven using the -X switch to enable full debug logging.
       > [ERROR]
       > [ERROR] For more information about the errors and possible solutions, please read the following articles:
       > [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
       >
       For full logs, run 'nix log /nix/store/cs23i4af8hzi55s06bd95hqx7gpm08ni-cryptomator-1.10.1.drv'.
```

Output of `nix log /nix/store/cs23i4af8hzi55s06bd95hqx7gpm08ni-cryptomator-1.10.1.drv`:

```
warning: The interpretation of store paths arguments ending in `.drv` recently changed. If this command is now failing try again with '/nix/store/cs23i4af8hzi55s06bd95hqx7gpm08ni-cryptomator-1.10.1.drv^*'
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/3j4k508x1sjjaknk4pbfkc8jkdm8n2bb-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
no configure script, doing nothing
@nix { "action": "setPhase", "phase": "buildPhase" }
building
[INFO] Scanning for projects...
[INFO]
[INFO] --------------------< org.cryptomator:cryptomator >---------------------
[INFO] Building Cryptomator Desktop App 1.10.1
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- license:2.2.0:add-third-party (add-third-party) @ cryptomator ---
[INFO] Writing third-party file to /build/source/target/generated-sources/license/THIRD-PARTY.txt
[INFO]
[INFO] --- resources:3.3.1:resources (default-resources) @ cryptomator ---
[INFO] Copying 165 resources from src/main/resources to target/classes
[INFO] Copying 1 resource from target/generated-sources/license to target/classes
[INFO]
[INFO] --- compiler:3.11.0:compile (default-compile) @ cryptomator ---
[WARNING] Can't extract module name from fuse-nio-adapter-3.0.0.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-linux-aarch64-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-win-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-linux-amd64-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-api-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-mac-0.5.1.jar: Unsupported major.minor version 64.0
[WARNING] Can't extract module name from jfuse-0.5.1.jar: Unsupported major.minor version 64.0
[INFO] Required filename-based automodules detected: [commons-cli-1.5.0.jar, javax.inject-1.jar]. Please don't publish this project to a public artifact repository!
[INFO] Changes detected - recompiling the module! :source
[INFO] Compiling 322 source files with javac [debug release 20 module-path] to target/classes
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.105 s
[INFO] Finished at: 2023-09-28T13:36:53Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project cryptomator: Fatal error compiling: error: release version 20 not supported -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```
My system: 

 ```
nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.5.5, NixOS, 23.11 (Tapir), 23.11.20230925.6500b45`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.0`
 - channels(root): `"nixos"`
 - channels(nixie): `""`
 - nixpkgs: `/nix/store/i6hq63805y784dml4j8pgkrz3lryxzld-source`
```

--- a/pkgs/tools/security/cryptomator/default.nix
+++ b/pkgs/tools/security/cryptomator/default.nix
@@ -13,17 +13,17 @@ in
 assert stdenv.isLinux; # better than `called with unexpected argument 'enableJavaFX'`
 mavenJdk.buildMavenPackage rec {
   pname = "cryptomator";
-  version = "1.9.4";
+  version = "1.10.1";
 
   src = fetchFromGitHub {
     owner = "cryptomator";
     repo = "cryptomator";
     rev = version;
-    hash = "sha256-63UXn1ejL/wDx6S2lugwwthu+C+vJovPypgM0iak78I=";
+    hash = "sha256-xhj7RUurBRq9ZIDAlcq7KyYGnLqc+vTjaf2VMNStpVQ";
   };
 
   mvnParameters = "-Dmaven.test.skip=true";
-  mvnHash = "sha256-7gv++Pc+wqmVYaAMgHhSy7xwChfVBgpDFxExzu3bXO0=";
+  mvnHash = "sha256-XAIwKn8wMqILMQbg9wM4kHAaRSGWQaBx9AXQyJuUO5k=";
 
   preBuild = ''
     VERSION=${version}

